ui/src/components/btn/__tests__/use-btn.spec.js

```diff
@@ -1,187 +1,513 @@
+import { mount } from '@cypress/vue'
+import { createMemoryHistory, createRouter } from 'vue-router'
+import { alignMap, alignValues } from '../../../composables/private/use-align.js'
+import QBtn from '../QBtn.js'
+import { btnPadding as paddingMap } from '../use-btn.js'
+
+const defaultOptions = {
+  label: 'simple Btn'
+}
+
+const typesValues = [ 'button', 'submit', 'reset' ]
+
+const paddingValues = Object.keys(paddingMap)
+
+function mountQBtn (options = {}) {
+  // Setup options object
+  options.global = options.global || {}
+  options.global.plugins = options.global.plugins || []
+
+  options.props = {
+    ...defaultOptions,
+    ...options.props
+  }
+
+  if (!options.router) {
+    options.router = createRouter({
+      routes: [],
+      history: createMemoryHistory()
+    })
+  }
+
+  // Add router plugin
+  options.global.plugins.push({
+    install (app) {
+      app.use(options.router)
+    }
+  })
+
+  return mount(QBtn, options)
+}
+
 describe('use-btn API', () => {
   describe('Props', () => {
     describe('Category: behavior|state', () => {
       describe('(prop): loading', () => {
-        it.skip(' ', () => {
-          //
+        it('should render a button with "loading" slot when "loading" prop is set to true', () => {
+          mountQBtn({
+            props: {
+              loading: true
+            }
+          })
+
+          cy.get('.q-btn')
+            .get('.absolute-full.flex.flex-center').should('exist')
```

Comment:
Searching for the spinner is a better indicator of the loading state
Remember that `.should('exist')` is implied by the `.get` command: it will keep retrying until it finds the selector you provided, or fail if it cannot find it

Apparently we're using it for other tests too, but it's not actually needed, we may remove it in a future refactor